utils/navigator_msgs/srv/ActivationSwitch.srv

```diff
@@ -0,0 +1,8 @@
+# Set to true to request activation of an idling process
```

Comment:
Oh you're right, this is unnecessary. I'll change it on my next PR since this works.